crates/bevy_ecs/src/query/access.rs

```diff
@@ -647,6 +647,16 @@ impl<T: SparseSetIndex> FilteredAccessSet<T> {
             .extend(filtered_access_set.filtered_accesses);
     }
 
+    /// Marks the set as reading all T.
```

Comment:
```suggestion
    /// Marks the set as reading all components that match the type `T`.
```

Reply:
Hmm, maybe this is incorrect actually. Regardless, I think we should be clearer here about what this does.